src/components/CustomInput/index.tsx

```diff
@@ -1,53 +1,77 @@
-import { Input } from '@openware/components';
+import { InputGroup, FormControl } from 'react-bootstrap';
 import * as React from 'react';
 
 export interface CustomInputProps {
     type: string;
     label: string;
     defaultLabel: string;
-    handleChangeInput: (value: string) => void;
+    handleChangeInput?: (value: string) => void;
     inputValue: string;
-    handleFocusInput: () => void;
-    classNameLabel: string;
-    classNameInput: string;
+    handleFocusInput?: () => void;
+    classNameLabel?: string;
+    classNameInput?: string;
     placeholder: string;
     autoFocus?: boolean;
     onKeyPress?: (event: React.KeyboardEvent<HTMLInputElement>) => void;
+    readOnly?: boolean;
+    id?: string;
+    handleClick?: ((event: React.MouseEvent<HTMLInputElement, MouseEvent>) => void) | undefined;
```

Comment:
handleClick`?`: ..... | `undefined` is duplicating, please keep just handleClick?: